internal/dotnet/properties/properties_test.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2022 OpenSSF Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package properties
+
+import (
+	"testing"
+)
+
+func TestIsValidFixedVersion(t *testing.T) {
+	t.Parallel()
+	tests := []struct {
+		name    string
+		version string
+		isFixed bool
+	}{
+		{"fixed version", "10.1.1", true},
+		{"fixed beta version", "10.1.1-beta", true},
```

Comment:
according to [your link in the regex comment](https://learn.microsoft.com/en-us/nuget/concepts/package-versioning?tabs=semver20sort#version-ranges), this is equivalent to "Minimum version, inclusive"?

so this conflicts with some of your tests below?

Reply:
fixed this by allowing only "Exact version match" using [] brackets.